### DISCUSSION TITLE
How can I use the bodymiscale integration (setup from gui) wiht multiple persons

### DISCUSSION THREAD

**Discussion by sorryusernameisalreadytaken** [dckiller51/bodymiscale#83] [Q&A]
hey there.

I wanna use this integration with multiple persons in my house. Is this possible?

**Comment by sorryusernameisalreadytaken** [comment]
Can nobody one help to answer this????

@dckiller51 how did you use this?

**Reply by dckiller51** [reply]
Impossible 1 user per card.

**Comment by 18rrs** [comment]
Just add integration for each person. I use it for 3.

**Reply by Virt1080** [reply]
Necro I know but do all 3 people use the same sensor? how does the system differentiate?

**Reply by Phoenix-DH** [reply]
@Virt1080 
See here:
https://github.com/dckiller51/bodymiscale/issues/160

**Comment by Phoenix-DH** [comment]
@18rrs 
How is this possible with one scale?

I thought you will only habe one sensor.
